docs/rfcs/0001_organizations.md

```diff
@@ -0,0 +1,98 @@
+# Organizations
+
+**Status:** Approved, waiting for implementation
+
+An organization encompasses a group of people working together to integrate many studies.
+
+## Why
+
+The concept of an an Organization arises from the need to manage groupings of studies and people with similar tooling. With an Organization, an institution or data coordinating center can establish a workspace for itself and administer its own studies and personnel. An Organization is intended to separate administrative needs from the data collection process.
+
+## How
+
+An Organization encompasses two important entities: studies and users. An Organization owns many studies where each study is related strictly to one Organization. An Organization also contains many users although each user may be a member of many Organizations.
```

Comment:
> An Organization encompasses two important entities: studies and users. 

Is storage backends not included here because it will be added in another RFC?

Reply:
Also wondering if we should support templates per organization or per study?

Reply:
Storage backends, templates, and whatever else are future propositions whereas studies and users are things we have to deal with at implementation time for Organizations. Those details can be spec'd in their own docs.